library/src/morpheus-master/js/src/sakai.morpheus.skipnav.js

```diff
@@ -11,7 +11,21 @@ var setupSkipNav = function(){
 };
 
 $PBJQ( document ).ready(function() {
-	
+        /*global $PBJQ, MorpheusViewportHelper*/
+        /*eslint no-undef: "error"*/
```

Comment:
Please remove this comment. Thanks.

Reply:
Unfortunately, when I remove that comment, then Codacy flags three errors, citing $PBJQ and MorpheusViewportHelper as undeclared. Should I add these back for these variables to be declared as globals so that Codacy check will pass?